res/qml/LibraryControl.qml

```diff
@@ -0,0 +1,188 @@
+import Mixxx 0.1 as Mixxx
+import QtQuick 2.12
+
+Item {
+    id: root
+
+    property alias focusWidget: focusedWidgetControl.value
+
+    signal moveSelection(int offset)
+    signal loadSelectedTrack(string group, bool play)
+    signal loadSelectedTrackIntoNextAvailableDeck(bool play)
+
+    FocusedWidgetControl {
+        id: focusedWidgetControl
+
+        Component.onCompleted: this.value = FocusedWidgetControl.WidgetKind.LibraryView
+    }
+
+    Mixxx.ControlProxy {
+        group: "[Library]"
+        key: "GoToItem"
+        onValueChanged: {
+            if (value != 0 && root.focusWidget == FocusedWidgetControl.WidgetKind.LibraryView)
+                root.loadSelectedTrackIntoNextAvailableDeck(false);
+
+        }
+    }
+
+    Mixxx.ControlProxy {
+        group: "[Playlist]"
+        key: "LoadSelectedIntoFirstStopped"
+        onValueChanged: {
+            if (value != 0 && root.focusWidget == FocusedWidgetControl.WidgetKind.LibraryView)
+                root.loadSelectedTrackIntoNextAvailableDeck(false);
+
+        }
+    }
+
+    Mixxx.ControlProxy {
+        group: "[Playlist]"
+        key: "SelectTrackKnob"
+        onValueChanged: {
+            if (value != 0) {
+                root.focusWidget = 3;
+                root.moveSelection(value);
+            }
+        }
+    }
+
+    Mixxx.ControlProxy {
+        group: "[Playlist]"
+        key: "SelectPrevTrack"
+        onValueChanged: {
+            if (value != 0) {
+                root.focusWidget = 3;
+                root.moveSelection(-1);
+            }
+        }
```

Comment:
Wondering if we should introduce another utility function for this?
Something like
```javascript
const keyUpGuard = function (binding) {
    return Qt.binding(function (value) {
        if (value !=0) {
            binding(value);
        }
    }
}
```
used like
```javascript
onValueChanged: keyUpGuard(Qt.binding(() => {
    root.focusWidget = 3;
    root.moveSelection(-1);
}));
```
just an idea, probably doesn't justify the complexity.

Reply:
Hmm, I think the decreased readability is not work the few lines that we can save that way.